### PR TITLE
Updated build pipeline to enable policheck

### PR DESCRIPTION
**Pull request by bozturkMSFT**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ pr:
 variables:
   # Cannot use key:value syntax in root defined variables
   - name: _TeamName
-    value: DotNetCore
+    value: dotnet-core-acquisition
```
Comment (donJoseLuis) [inline azure-pipelines.yml]: Question: is dotnet-core-acquisition the proper name for the templating repo?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -128,9 +128,29 @@ stages:
...
+#        enable: true
+#        continueOnError: false
+#        params: ' -SourceToolsList @("policheck","credscan")
+#        -TsaInstanceURL $(_TsaInstanceURL)
+#        -TsaProjectName $(_TsaProjectName)
+#        -TsaNotificationEmail $(_TsaNotificationEmail)
+#        -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
+#        -TsaBugAreaPath $(_TsaBugAreaPath)
+#        -TsaIterationPath $(_TsaIterationPath)
+#        -TsaRepositoryName "$(TsaRepositoryName)"
```
Comment (donJoseLuis) [inline azure-pipelines.yml]: should this variable just be string "templating"?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -128,9 +128,29 @@ stages:
...
+#        -TsaCodebaseName "$(TsaCodebaseName)"
```
Comment (donJoseLuis) [inline azure-pipelines.yml]: should this variable just be string "templating"?

File: azure-pipelines.yml (hunk 1 above)
Comment (bozturkMSFT) [reply azure-pipelines.yml]: I used the same values as runtime team as requested in the issue #2324. Or did I misunderstand?

This is what runtime uses: https://github.com/dotnet/runtime/blob/49118f3385f12f87ec916ec3398a429fa3d11c69/eng/pipelines/runtime-official.yml#L27

File: azure-pipelines.yml (hunk 2 above)
Comment (bozturkMSFT) [reply azure-pipelines.yml]: Sample PR in websdk is using string literals. However, I again followed runtime repo and did as they did: https://github.com/dotnet/runtime/blob/385121f62de417c2a2416add568b34880b4c5ea6/eng/pipelines/official/stages/publish.yml#L51

I don't really know what is the "correct" way of doing this.

Comment (donJoseLuis) [reply azure-pipelines.yml]: I trust you did the right thing. I recall I reviewed Vijay's PR & saw they hard coded the repository's name instead.  All good.

File: azure-pipelines.yml (hunk 3 above)
Comment (donJoseLuis) [reply azure-pipelines.yml]: Already discussed in previous comment

File: azure-pipelines.yml (hunk 1 above)
Comment (donJoseLuis) [reply azure-pipelines.yml]: Looks like you're clear on the issue, I defer to you.

**Comment by bozturkMSFT** [comment]
PR Validation builds do not execute SDL Validation.
Policheck successfully ran in the internal build pipeline here: https://dev.azure.com/dnceng/internal/_build/results?buildId=567824&view=results

**Pull request merged**